src/status_im/utils/config.cljs

```diff
@@ -18,12 +18,16 @@
 ;; NOTE(oskarth): Feature flag deprecation lifecycles. We want to make sure
 ;; flags stay up to date and are removed once behavior introduced is stable.
 
-(goog-define INFURA_TOKEN "800c641949d64d768a5070a1b0511938")
+(goog-define POKT_TOKEN "3ef2018191814b7e1009b8d9")
 (goog-define OPENSEA_API_KEY "")
 
-(def mainnet-rpc-url (str "https://mainnet.infura.io/v3/" INFURA_TOKEN))
-(def testnet-rpc-url (str "https://ropsten.infura.io/v3/" INFURA_TOKEN))
-(def goerli-rpc-url  (str "https://goerli.infura.io/v3/" INFURA_TOKEN))
+(def mainnet-rpc-url
+  (str "https://eth-archival.gateway.pokt.network/v1/lb/" POKT_TOKEN))
+
+(def testnet-rpc-url
```

Comment:
ropsten is still needed?

Reply:
@flexsurfer no but I will remove in a separate PR anyway